Integrations/integration-Ipinfo.yml

```diff
@@ -2,6 +2,7 @@ commonfields:
   id: ipinfo
   version: -1
 name: ipinfo
+releaseNotes: Added ouputs  
```

Comment:
Release notes should be more descriptive since users read this when we releasing content.